vanpy/videos/oss-continuity-and-change.json

```diff
@@ -5,6 +5,14 @@
     {
       "label": "Event Web Site",
       "url": "http://www.vanpyday.com"
+    },
+    {
+      "label": "registrasion",
```

Comment:
I thought the same, but...

https://pypi.python.org/pypi/registrasion
**Registra** (tion for Sympo) **sion**. A conference registration app for Django, letting conferences big and small sell tickets from within Symposion.